### ISSUE TITLE
Templater prompts with button 'remove' option

### ISSUE THREAD

**Issue by ConnorMeyers** [shabegom/buttons#57]
When templater prompts are mixed with a button to replace the templates and also set to remove, the button doesn't get removed. (likely because the template prompt reinserts it)

```
<% tp.system.prompt("Enter text.") %>

button
name Finish
type command
action Templater: Replace templates in the active file
remove true
```

**Comment by shabegom** [comment]
Yep! I was also having this problem. I'll look into it. Thanks for reporting!

**Comment by shabegom** [comment]
I think this will be fixed in the next release. Just working on another bug and then will push out a new version. Basically, I'm delaying the remove a little bit longer so it happens after Templater has finished processing the note.

**Comment by shabegom** [comment]
Should be fixed in 0.4.12
